scripts/openmc-plot-mesh-tally

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
+
+"""Python script to plot tally data generated by OpenMC."""
+
+import os
+import sys
+import argparse
+import tkinter as tk
+import tkinter.filedialog as filedialog
+import tkinter.font as font
+import tkinter.messagebox as messagebox
+import tkinter.ttk as ttk
+
+import matplotlib
+matplotlib.use("TkAgg")
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
+from matplotlib.backends.backend_tkagg import NavigationToolbar2Tk
+from matplotlib.figure import Figure
+import matplotlib.pyplot as plt
+import numpy as np
+
+from openmc import StatePoint, MeshFilter
+
+
+class MeshPlotter(tk.Frame):
+    def __init__(self, parent, filename):
+        tk.Frame.__init__(self, parent)
+
+        self.labels = {'Cell': 'Cell:', 'Cellborn': 'Cell born:',
```

Comment:
optional for readability, better to define each key-value in separate line:
```
self.labels = {
    'Cell': 'Cell:',
    'Cellborn': 'Cell born:',
    'Surface': 'Surface:'
...
}```

Reply:
Fixed in #1424